python-lib/wordcloud_visualizer.py

```diff
@@ -122,7 +116,9 @@ def __init__(
 
     def _color_func(self, word, font_size, position, orientation, random_state=None, **kwargs) -> AnyStr:
```

Comment:
random_state doesn't seem to be used

Reply:
removed, didn't need other arguments as well so removed them